core/src/cats/create_bareos_database.in

```diff
@@ -104,16 +104,14 @@ case ${db_type} in
    postgresql)
       # use SQL_ASCII to be able to put any filename into
       # the database even those created with unusual character sets
-      ENCODING="ENCODING 'SQL_ASCII' LC_COLLATE 'C' LC_CTYPE 'C'"
-
       PGOPTIONS='--client-min-messages=warning' psql -f - -d template1 $* << END-OF-DATA
 \set ON_ERROR_STOP on
-CREATE DATABASE ${db_name} $ENCODING TEMPLATE template0;
+CREATE DATABASE ${db_name} ENCODING 'SQL_ASCII' LC_COLLATE 'C' LC_CTYPE 'C' TEMPLATE template0;
 ALTER DATABASE ${db_name} SET datestyle TO 'ISO, YMD';
 END-OF-DATA
       retval=$?
 
-      if PGOPTIONS='--client-min-messages=warning' psql -l ${dbname} $* | grep " ${db_name}.*SQL_ASCII" >/dev/null; then
+      if PGOPTIONS='--client-min-messages=warning' psql --list --tuples-only --no-align $* | grep "^bareos|.*|SQL_ASCII|C|C" >/dev/null; then
```

Comment:
that will not work if the databasename is different from "bareos"

Reply:
true I forget the ${db_name}